akka-stream/src/main/scala/akka/stream/javadsl/Flow.scala

```diff
@@ -2589,6 +2588,23 @@ final class Flow[In, Out, Mat](delegate: scaladsl.Flow[In, Out, Mat]) extends Gr
   def alsoTo(that: Graph[SinkShape[Out], _]): javadsl.Flow[In, Out, Mat] =
     new Flow(delegate.alsoTo(that))
 
+  /**
+   * Attaches the given [[Sink]]s to this [[Flow]], meaning that elements that passes
+   * through will also be sent to the [[Sink]].
+   *
+   * It is similar to [[#wireTap]] but will backpressure instead of dropping elements when the given [[Sink]] is not ready.
+   *
+   * '''Emits when''' element is available and demand exists both from the Sink and the downstream.
+   *
+   * '''Backpressures when''' downstream or Sink backpressures
+   *
+   * '''Completes when''' upstream completes
+   *
+   * '''Cancels when''' downstream or Sink cancels
+   */
+  def alsoToAll(those: java.util.List[Graph[SinkShape[Out], _]]): javadsl.Flow[In, Out, Mat] =
```

Comment:
I think the Java signatures should also be vararg rather than take a collection/list